db/db.go

```diff
@@ -78,6 +79,9 @@ func (db *DB) Exec(sql_or_name string, args ...interface{}) error {
 		return err
 	}
 
+	if os.Getenv("SHIELD_DB_TRACE") != "" {
+		fmt.Fprintf(os.Stdout, "DB is execing '%s' ", sql_or_name)
+	}
```

Comment:
Let's drop the debugging, and put in a ticket to re-add the behavior in a separate changeset.